oneclick_monolith_install/bootstrap-env-playbook.yml

```diff
@@ -0,0 +1,67 @@
+# Sets up a fresh environments directory
+# Run using `ansible-playbook --connection=local bootstrap-env-config.yml --extra-vars "env_name=$env_name"`
+
+- hosts: 127.0.0.1
+  connection: local
+
+  vars:
+    install_config:
+      - '{{env_name}}'
+      - '{{site_host}}'
+      - '{{server_inventory_name}}'
+      - '{{server_host_name}}'
+      - '{{ssh_public_key}}'
+      - '{{ssh_username}}'
+    config_is_not_valid: '{{"" in install_config}}'
+    envs_path: "{{ ansible_env.HOME }}/environments"
+    env_path: "{{envs_path}}/{{ env_name }}"
+  tasks:
+    - debug:
+        msg: "Invalid installation configuration: {{install_config}}. Please define all the variables in your install-config.yml file."
+      when: config_is_not_valid
+    - meta: end_play
+      when: config_is_not_valid
+    - name: Create environments directory
+      file: path={{env_path}} state=directory
+    - name: Create env's users directory
+      file: path={{envs_path}}/_users state=directory
+    - name: Create ssh-public-keys directory
+      file: path={{envs_path}}/_authorized_keys state=directory
+    - name: Copy templated env files
+      template:
+        src: "./sample-environment/{{item}}.j2"
```

Comment:
Note that this kind of env bootstrap exists via [commcare-cloud-bootstrap](https://github.com/dimagi/commcare-cloud/tree/master/commcare-cloud-bootstrap) but it uses very specific format and requires a lot more details from the user as it fits into a different workflow.

I have also observed that there is https://github.com/dimagi/sample-environment but that repo is not getting upto date when something is modified in the commcare-cloud repo so it has a tendency to get outdated.